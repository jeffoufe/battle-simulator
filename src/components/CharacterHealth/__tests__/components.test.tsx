import React from 'react';
import { shallow } from 'enzyme';
import { 
    CharacterHealthContainer,
    CharacterHealthIndicatorContainer,
    CharacterHealthIndicator,
    CharacterHealthPoints
} from '../components'

describe('CharacterHealth Components', () => {
    it('CharacterHealthContainer', () => {
        const wrapper = shallow(
            <CharacterHealthContainer />
        );
        expect(wrapper).toMatchSnapshot();
    })

    it('CharacterHealthIndicatorContainer', () => {
        const wrapper = shallow(
            <CharacterHealthIndicatorContainer />
        );
        expect(wrapper).toMatchSnapshot();
    })

    it('CharacterHealthIndicator', () => {
        const wrapper = shallow(
            <CharacterHealthIndicator health={50} />
        );
        expect(wrapper).toMatchSnapshot();
    })

    it('CharacterHealthPoints', () => {
        const wrapper = shallow(
            <CharacterHealthPoints>
                5
            </CharacterHealthPoints>
        );
        expect(wrapper).toMatchSnapshot();
    })
})