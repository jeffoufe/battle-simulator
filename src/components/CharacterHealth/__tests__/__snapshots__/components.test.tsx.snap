// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterHealth Components CharacterHealthContainer 1`] = `
<div
  className="sc-AxjAm eYcNGP"
/>
`;

exports[`CharacterHealth Components CharacterHealthIndicator 1`] = `
<div
  className="sc-AxiKw ldBGmk"
/>
`;

exports[`CharacterHealth Components CharacterHealthIndicatorContainer 1`] = `
<div
  className="sc-AxirZ chhwUe"
/>
`;

exports[`CharacterHealth Components CharacterHealthPoints 1`] = `
<span
  className="sc-AxhCb gYqkZJ"
>
  5
</span>
`;
